Automattic/wp-calypso
# Add ReaderAuthorLink and ReaderSiteStreamLink components

`ReaderAuthorLink` and `ReaderSiteStreamLink` are new components that accept a post object and then figure out how to link to an author's site, or the feed's stream respectively.

They also record relevant stats when the link is clicked.

The old `site-link` currently exists inside `client/reader` and is removed with this PR. It's not currently in use.

### To test

Neither component is yet in use, but you can see them in action on the app components example page:

http://calypso.localhost:3000/devdocs/app-components

Test live: https://calypso.live/?branch=add/reader/author-and-site-link-components
